src/role_harvester.js

```diff
@@ -31,7 +31,20 @@ roles.harvester.settings = {
   maxLayoutAmount: 6,
 };
 roles.harvester.updateSettings = function(room) {
-  if (room.storage && room.storage.my && room.storage.store.energy > config.creep.energyFromStorageThreshold && room.energyAvailable > 350 && !room.memory.misplacedSpawn) {
+  // If the energy average is lower than 350 for a longer period at least one low level harvester will be spawned.
+  // Fix for the following problem:
+  // - no room is able to send a nextroomer
+  // - all creeps are dead
+  // - energyAvailable is higher than 300 but lower than 350 for a longer time
+  if (room.memory.energyStats.average <= 350) {
+    // Layoutcost minimum: layout 250 -> 250
+    return {
+      layoutString: 'MWC',
+      amount: [2, 1, 1],
+      maxLayoutAmount: 6,
+    };
+  } else if (room.storage && room.storage.my && room.storage.store.energy > config.creep.energyFromStorageThreshold && room.energyAvailable > 350 && !room.memory.misplacedSpawn) {
+    // Layoutcost minimum: prefix 250 + layout 100 -> 350
```

Comment:
Fine, let's keep it like this.

But, the if clause explicitly states, that `room.energyAvailable` needs to be `> 350` otherwise the default settings are used.

Reply:
Mhh you're right but I had this problem sometimes. I will check if this problem persists!